Python/Product/EnvironmentsList/EnvironmentPathsExtension.xaml

```diff
@@ -358,35 +362,31 @@
+                                  Margin="6 9 6 6">
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="auto" />
+                                    <ColumnDefinition />
+                                </Grid.ColumnDefinitions>
+                                <ContentControl Grid.Column="0"
+                                                    Margin="3 0"
+                                                    Style="{StaticResource HelpImage}"
+                                                    IsTabStop="False"/>
+                                <vsui:HyperlinkButton
+                                    x:Name="SupportUrl"
+                                    Grid.Column="1"
+                                    HorizontalAlignment="Left"
+                                    Content="{x:Static l:Resources.EnvironmentPathsExtensionSupportInformationLabel}"
                                     Command="{x:Static l:EnvironmentPathsExtension.OpenInBrowser}"
                                     CommandParameter="{Binding SupportUrl,Mode=OneWay}"
                                     ToolTip="{Binding SupportUrl,Mode=OneWay}"
-                                    Style="{StaticResource NavigationButton}"
-                                    Margin="6 9 6 4"
-                                    HorizontalAlignment="Left"
                                     AutomationProperties.Name="{x:Static l:Resources.EnvironmentPathsExtensionSupportInformationLabel}"
                                     AutomationProperties.HelpText="{Binding SupportUrl,Mode=OneWay}">
-                                <Grid Background="Transparent">
-                                    <Grid.ColumnDefinitions>
-                                        <ColumnDefinition Width="auto" />
-                                        <ColumnDefinition />
-                                    </Grid.ColumnDefinitions>
-                                    <ContentControl Grid.Column="0"
-                                                    Margin="3 0"
-                                                    Style="{StaticResource HelpImage}"
-                                                    IsTabStop="False"/>
-                                    <Label Grid.Column="1" Style="{StaticResource FileNameLabel}"
-                                           Foreground="{Binding Foreground,RelativeSource={RelativeSource AncestorType=Button}}"
-                                           Content="{x:Static l:Resources.EnvironmentPathsExtensionSupportInformationLabel}"/>
-                                </Grid>
-                            </Button>
+                                </vsui:HyperlinkButton>
+                            </Grid>
                         </StackPanel>
 
                         <DataTemplate.Triggers>
-                            <DataTrigger Binding="{Binding SupportUrl,Mode=OneWay}" Value="">
```

Comment:
I think we need this, otherwise it will look weird if support url is empty. You can make it control visibility of your outer grid, ie move the x:Name="SupportUrl" from the button to its parent grid.

Reply:
agreed.

Reply:
now hidden
![image](https://user-images.githubusercontent.com/1946977/70741251-b80eb100-1ccf-11ea-8f0f-784e8848ea2f.png)